Frameworks/include/CGGradientInternal.h

```diff
@@ -17,8 +17,8 @@
 
 #pragma once
 
-#include "CoreGraphics/CGGradient.h"
-#include "CoreGraphicsInternal.h"
+#import <CoreGraphics/CGGradient.h>
+#import "CoreGraphicsInternal.h"
 #include <objc/runtime.h>
 
 class __CGGradient : private objc_object {
```

Comment:
not part of this scope.
Yep does definitely needs to be updated 

---
In reply to: [92450352](https://github.com/Microsoft/WinObjC/pull/1535#discussion_r92450352) [](ancestors = 92450352)